service/routes.py

```diff
@@ -1,12 +1,17 @@
 """
-My Service
+Product Service
 
-Describe what your service does here
+Paths -- RESTful
 """
```

Comment:
we should add some description in here
eg:
"""
Pet Store Service
Paths:
GET /pets - Returns a list all of the Pets
GET /pets/{id} - Returns the Pet with a given id number
POST /pets - creates a new Pet record in the database
PUT /pets/{id} - updates a Pet record in the database
DELETE /pets/{id} - deletes a Pet record in the database
"""